app/mailers/mailer.rb

```diff
@@ -103,7 +103,7 @@ def feedback(user, feedback)
     @user = user
     @feedback = feedback
 
-    mail to: FoodsoftConfig[:notification][:error_recipients],
+    mail to: FoodsoftConfig[:notification][:feedback_recipients],
```

Comment:
Ideally this would go in [FoodsoftConfig#get_default_config](https://github.com/foodcoops/foodsoft/blob/master/lib/foodsoft_config.rb#L254). But as the fallback is a value depending on another configuration item, it won't work.

What about:
```ruby
def feedback(user, feedback)
  #...
  mail to: feedback_recipients,
  # ...
end

private

def feedback_recipients
  # use the (new) feedback_recipients option, but fallback to error_recipients for backwards compatibility
  FoodsoftConfig[:notification][:feedback_recipients] || FoodsoftConfig[:notification][:error_recipients]
end
```
```

Reply:
Thanks for your feedback. I added the priavte method and squashed the commit.